backend/supabase/functions/_shared/workspace-quota-manager.ts

```
@@ -0,0 +1,502 @@
+          return {
+            allowed: false,
+            reason: `Project limit exceeded (${usage.projects_count}/${quota.max_projects})`,
+            current_usage: { projects_count: usage.projects_count },
+            quota_limits: { max_projects: quota.max_projects },
+            usage_percentage: (usage.projects_count / quota.max_projects) * 100
+          };
+        }
+        break;
+
+      case 'api_call':
+        const dailyLimit = quota.max_api_calls_per_day || 1000;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)